src/test/regress/expected/multi_extension.out

```diff
@@ -120,17 +149,256 @@ ALTER EXTENSION citus UPDATE TO '9.0-2';
+                 | function worker_execute_sql_task(bigint,integer,text,boolean)
+                 | function worker_fetch_foreign_file(text,text,bigint,text[],integer[])
+                 | function worker_fetch_partition_file(bigint,integer,integer,integer,text,integer)
+                 | function worker_hash("any")
+                 | function worker_hash_partition_table(bigint,integer,text,text,oid,anyarray)
+                 | function worker_merge_files_and_run_query(bigint,integer,text,text)
+                 | function worker_merge_files_into_table(bigint,integer,text[],text[])
+                 | function worker_partial_agg(oid,anyelement)
+                 | function worker_partial_agg_ffunc(internal)
+                 | function worker_partial_agg_sfunc(internal,oid,anyelement)
+                 | function worker_partition_query_result(text,text,integer,citus.distribution_type,text[],text[],boolean)
+                 | function worker_range_partition_table(bigint,integer,text,text,oid,anyarray)
+                 | function worker_repartition_cleanup(bigint)
+                 | schema citus
+                 | schema citus_internal
+                 | sequence pg_dist_colocationid_seq
+                 | sequence pg_dist_groupid_seq
+                 | sequence pg_dist_node_nodeid_seq
+                 | sequence pg_dist_placement_placementid_seq
+                 | sequence pg_dist_shardid_seq
+                 | table citus.pg_dist_object
+                 | table pg_dist_authinfo
+                 | table pg_dist_colocation
+                 | table pg_dist_local_group
+                 | table pg_dist_node
+                 | table pg_dist_node_metadata
+                 | table pg_dist_partition
+                 | table pg_dist_placement
+                 | table pg_dist_poolinfo
+                 | table pg_dist_rebalance_strategy
+                 | table pg_dist_shard
+                 | table pg_dist_transaction
+                 | type citus.distribution_type
+                 | type citus.shard_transfer_mode
+                 | type citus_copy_format
+                 | type noderole
+                 | view citus_dist_stat_activity
+                 | view citus_lock_waits
+                 | view citus_shard_indexes_on_worker
+                 | view citus_shards_on_worker
+                 | view citus_stat_statements
+                 | view citus_worker_stat_activity
+                 | view pg_dist_shard_placement
+(188 rows)
+
+-- Test downgrade to 9.2-2 from 9.2-4
 ALTER EXTENSION citus UPDATE TO '9.2-4';
-/*
- * As we mistakenly bumped schema version to 9.3-1 (in previous
- * release), we support updating citus schema from 9.3-1 to 9.2-4,
- * but we do not support explicitly updating it to to 9.3-1.
- * Hence below update (to 9.3-1) command should fail.
- */
-ALTER EXTENSION citus UPDATE TO '9.3-1';
-ERROR:  extension "citus" has no update path from version "9.2-4" to version "9.3-1"
```

Comment:
I guess this test is removed now, because as a sideffect of these changes updating to `9.3-1` is possible now, i.e. by downgrading from `9.2-4` to `9.2-2` and then to `9.3-1`. This seems fine to me. @marcocitus @onderkalaci I believe you introduced these `9.3-1` related scripts. Do you agree that it's fine?

Reply:
No, not really. There is no way to upgrade/downgrade to `9.3-1`. The only migration script on this branch that has `9.3-1` in the name is `citus--9.3-1--9.2-4.sql`.

I removed the test because I could not decide on where to put said test. No matter where I place it, there is no migration path. Maybe I should reintroduce it after all the regular `ALTER EXTENSION citus UPDATE` queries.

Reply:
Ah ok, maybe simply put it at the start of all the upgrade scripts.